plugins/source/terraform/docs/index.md

```diff
@@ -39,13 +32,13 @@ CloudQuery currently supports LOCAL and S3 backends.
 ## S3 backend example
 
 ```yaml
-    config:
-      - name: myremote # s3 backend
-        backend: s3
-        bucket: "<terraform state bucket>"
-        key: "<terraform state key>"
-        region: us-east-1
-        role_arn: ""
+backends:
+  - name: myremote # s3 backend
+    s3:
+      bucket: "<terraform state bucket>"
+      key: "<terraform state key>"
+      region: us-east-1
+      role_arn: ""
```

Comment:
Thanks for updating the config! This example (and the local one above it) will also need to be updated now

Reply:
Fixed